.github/workflows/docker-build.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         run: |
           sudo rm -rf "/usr/local/share/boost"
           sudo rm -rf "$AGENT_TOOLSDIRECTORY"
-      - uses: actions/setup-python@v2
+      - uses: actions/setup-python@v2.3.3
```

Comment:
Let's use v4 instead. See https://github.com/actions/setup-python#basic-usage

Reply:
@vfdev-5 btw when i checked the latest version it was showing 2.3.3 at marketplace but yeah i have done it to v4 
![Screenshot from 2022-12-01 19-42-36](https://user-images.githubusercontent.com/87641376/205074722-fd07aed6-3aee-4685-a284-a44628ebeddb.png)

Reply:
v2.3.3 seems like a backport of some newest features, https://github.com/actions/setup-python/releases